examples/jenkins/Linux.Jenkinsfile

```diff
@@ -36,39 +36,36 @@ pipeline {
   stages {
     stage('Install Python Dependencies') {
       steps {
-        echo "Create ${env.ArtifactsFolder} Folder"
+        echo "Create Artifacts Folder"
         sh "mkdir ${env.ArtifactsFolder}"
         // Only the virtual environment needs to be installed at the system level
         echo "Install Python Virtual environments"
-        sh 'pip3 install -q -I virtualenv --user'
-        // Install the rest of the dependencies at the environment level and not the system level
-        withPythonEnv('python') {
+        sh 'pip3 install -q -I virtualenv'
+        // Install the rest of the dependencies
+        withPythonEnv('python3') {
           echo "Install Python requirements"
           sh 'pip3 install -U outsystems-pipeline'
         }
       }
     }
-    stage('Get and Deploy Latest Tags') {
+    stage('Get Latest Tags') {
```

Comment:
Rename the stage to "Get and Deploy Latest Tags" to keep consistency with Windows version.

Reply:
Stage renamed